packages/tinacms/src/components/TinaProvider.tsx

```diff
@@ -29,18 +29,20 @@ export interface TinaProviderProps {
   cms: TinaCMS
   hidden?: boolean
   position?: SidebarPosition
+  styled?: boolean
 }
 
 export const TinaProvider: React.FC<TinaProviderProps> = ({
   cms,
   children,
   hidden,
   position,
+  styled = true,
 }) => {
   return (
     <CMSContext.Provider value={cms}>
       <ModalProvider>
-        <GlobalStyles />
+        <StyleProvider active={styled} />
```

Comment:
I'm just realizing the `Provider` suffix is a bit misleading.

The `Provider` convention is a React Context thing. i.e.

```
const Style = React.createContext(...)

<Style.Provider><App /></Style.Provider>
```

The `GlobalStyles` name better fits the naming conventions for this use case.

Maybe the old `GlobalStyles` could be renamed to `GlobalCSS` to distinguish it from the `FontLoader`

Reply:
Note that this also goes for the Toolbar, but no need to address that now.

Reply:
we could call `TinaGlobalStyles` --> `TinaTheme`

Reply:
That works 👌

Reply:
or `StyleProvider` --> `TinaTheme`....might be nice

Reply:
```
Theme
    Fonts
    Styles
```